packages/ckeditor5-ckbox/docs/features/ckbox.md

```diff
@@ -30,12 +30,12 @@ As a full-fledged file manager, CKBox also replaces the basic CKEditor&nbsp;5 im
 * Enables drag & drop uploads of images and other files.
 * Transforms the _Image_ toolbar button, allowing the user to quickly upload and insert an image without opening the CKBox UI.
 * Adds a separate dedicated toolbar button to open the CKBox UI to manage and reuse uploaded files.
-* Provides basic editing capabilities, like crop, resize, rotate, and flip.
+* Provides {@link @ckbox features/images/editing basic editing capabilities}, like crop, resize, rotate, and flip right from the {@link features/images-overview#image-contextual-toolbar image contextual toolbar}.
```

Comment:
It's a nitpick, but "provides" sounds a little redundant here to me.

Reply:
Let's keep it, makes perfect sense information-wise.